### PR TITLE
Lizzards now dislike Toxic , fried and JUNKFOOD food but no longer dislike grain types

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Lizards now dislike those toxic foods and cant stand the horrors of oil food as well as junkfood
But have overcome their fear of rice

## Why It's Good For The Game

Makes lizards be able to eat like 8/10ths of the food in game that gets cooked

## Changelog
:cl:
balance: Lizard People now dislike toxic, fried and junk food, but now can eat a-grain types
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	gib_types = list(/obj/effect/gibspawner/lizard, /obj/effect/gibspawner/lizard/bodypartless)
 	skinned_type = /obj/item/stack/sheet/animalhide/lizard
 	exotic_bloodtype = "L"
-	disliked_food = GRAIN | DAIRY
+	disliked_food = TOXIC | FRIED
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Dude, their `toxic_food` bitfield already contains the `TOXIC` flag.
Look at `human/species.dm`, open your eyes.

**Review by Ghommie — DISMISSED** [review]
A lizardfolk player complaining about their species disliked food is no real lizardfolk player. Species shouldn't be purely cosmetic, nope. I dislike this change.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: This re-wrights over the other disliked food??

**Comment by necromanceranne** [comment]
They're obligate carnivores, that's why they hate anything but raw meat.

**Comment by Trilbyspaceclone** [comment]
@necromanceranne take devent tastes and then you hate meat and love flour and milk...

**Comment by necromanceranne** [comment]
Deviant tastes is deviant for a reason. That's why humans with it eat literal trash.

**Comment by Tupinambis** [comment]
Don't remove species flavor.

**Comment by Linzolle** [comment]
no

**Comment by Owai-Seek** [comment]
This would make it so lizards can't eat -anything- if there's a shitter chef frying random shit.

There are only a few things lizards cant eat and its fine.
Grain (Oat/Wheat)
Tortilla Foods (Considered Grain)
Potato Dishes (Maybe Considered Grain?)
Most Dairy (Ice Cream)

You can easily get around the restrictions by going to ghetto botany and growing plants, or buying fruit/monkey cube crates from cargo.

**Comment by SectoidDan** [comment]
There's two places where lizardpeople not being able to eat grains messes with balance: Lavaland, with its abundance of donkpockets for healing, and Security, who have their donuts. Yes, you can use a blender for the donkpockets and there is one in the mining base, but shelter capsules don't have blenders, and honestly that's a very gamey/cheesy way to get around the restrictions anyway.

>You can easily get around the restrictions by going to ghetto botany and growing plants, or buying fruit/monkey cube crates from cargo.

You can do that if a shitter chef is deep frying everything, too.

**Pull request closed without merging**